src/api/fixtures/LoanSearchFilters.js

```diff
@@ -68,10 +69,12 @@ export function getInputFilters(filters) {
 // Return filters that can be used in a loan search
 export function getSearchableFilters(filters) {
```

Comment:
This is the only method I'm currently using to create a loanQuery for the campaign pages.

Reply:
Addition of `loanTags` in the other methods don't appear to have any affect on the functionality of auto lending so I think they are ok.